source/fundamentals/transactions.txt

```diff
@@ -71,25 +72,42 @@ following table describes these methods:
      - | Starts a new transaction, configured with the given options, on
          this session. Returns an error if there is already
          a transaction in progress for the session. For more
-         information, see the :manual:`manual entry </reference/method/Session.startTransaction/>`.
+         information, see :manual:`startTransaction()
+         </reference/method/Session.startTransaction/>`  in the Server manual.
```

Comment:
S: I think these sentences would flow a little better when we say "see the <link> guide (or page) in the..." 
Applies to all "For more information..." instances

```suggestion
         information, see the :manual:`startTransaction()
         </reference/method/Session.startTransaction/>` guide in the Server manual.
```